Frameworks/Foundation/NSArray.mm

```diff
@@ -1031,8 +1031,10 @@ - (NSUInteger)indexOfObjectIdenticalTo:(id)anObject inRange:(NSRange)range {
                            [self count]];
     }
 
+    id objects[range.length];
+    [self getObjects:objects range:range];
     for (NSUInteger i = range.location; i < end; ++i) {
```

Comment:
Considered it but this was the path of least change, didn't think it made a substantial difference in readability or perf